### ISSUE TITLE
Error compling sass

### ISSUE THREAD

**Issue by mazikwyry** [Dogfalo/materialize#391]
I compile sass with compass (sass v 3.4.9) and have following error:

```
error sass/materialize.scss (Line 353 of sass/components/_global.scss: ".collection .active" failed to @extend ".blue-text.lighten-4".
The selector ".blue-text.lighten-4" was not found.
Use "@extend .blue-text.lighten-4 !optional" if the extend should be able to fail.
)
```

BTW. This library is amazing and very well done!! Congs +1

**Comment by mazzie14** [comment]
I get the same error in Codekit when compiling : 
```
Error: ".collection .active" failed to @extend ".blue-text.lighten-4".
       The selector ".blue-text.lighten-4" was not found.
       Use "@extend .blue-text.lighten-4 !optional" if the extend should be able to fail.
        on line 353 of /Library/WebServer/Documents/provo new/dist/sass/components/_global.scss
```

**Comment by mazikwyry** [comment]
I've followed tip from the error message `Use "@extend .blue-text.lighten-4 !optional" if the extend should be able to fail.` and added `!optional` in line `353` of `sass/components/_global.scss` and then woks fine.

**Comment by acburst** [comment]
It seems to be a typo. The most up to date version should be fine

**Comment by Dogfalo** [comment]
Fixed in 9e6a50841e68302bc2b6e3c79c3416b20c292208
